modules/Ecommerce/Banner/Requests/GetFeatureListRequest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Modules\Ecommerce\Banner\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class GetFeatureListRequest extends FormRequest
+{
+    /**
+     * Get the validation rules that apply to the request.
+     */
+    public function rules(): array
+    {
+        return [
+            'page' => ['nullable', 'integer', 'min:1'],
+            'per_page' => ['nullable', 'integer', 'min:1', 'max:100'],
+            'setting_page_id' => ['nullable', 'uuid', 'exists:setting_pages,id'],
+            'setting_page_type' => ['nullable', 'string', 'max:255'],
+            'is_active' => ['nullable', 'boolean'],
+            'search' => ['nullable', 'string', 'max:255'],
+            'title' => ['nullable', 'string', 'max:255'],
+            'description' => ['nullable', 'string', 'max:255'],
+            'active_only' => ['nullable', 'boolean'],
+            'inactive_only' => ['nullable', 'boolean'],
+            'created_from' => ['nullable', 'date'],
+            'created_to' => ['nullable', 'date|after_or_equal:created_from'],
```

Comment:
The validation rule for `created_to` has incorrect syntax. The pipe character should be inside quotes as part of the rule string, not separating rules.

```suggestion
            'created_to' => ['nullable', 'date', 'after_or_equal:created_from'],
```